test/models/katello/ostree_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'katello_test_helper'
+
+module Katello
+  class OstreeTest < ActiveSupport::TestCase
+    def setup
+
+      @ostree = FactoryBot.create(:katello_repository, :ostree, :with_product)
+    end
+
+    def test_created
+
+      assert @ostree
+    end
+  end
+end
```

Comment:
i'd probably be in favor of leaving tests out for now until we're on pulp 3.15 and other devs can record ostree vcrs